DeviceAdministration/Infrastructure.UnitTests/Infrastructure/DeviceLogicTests.cs

```diff
@@ -30,36 +31,39 @@ public class DeviceLogicTests
 
         public DeviceLogicTests()
         {
-            _iotHubRepositoryMock = new Mock<IIotHubRepository>();
-            _deviceRegistryCrudRepositoryMock = new Mock<IDeviceRegistryCrudRepository>();
-            _deviceRegistryListRepositoryMock = new Mock<IDeviceRegistryListRepository>();
-            _virtualDeviceStorageMock = new Mock<IVirtualDeviceStorage>();
-            _configProviderMock = new Mock<IConfigurationProvider>();
-            _securityKeyGeneratorMock = new Mock<ISecurityKeyGenerator>();
-            _deviceRulesLogicMock = new Mock<IDeviceRulesLogic>();
-            _deviceLogic = new DeviceLogic(_iotHubRepositoryMock.Object, _deviceRegistryCrudRepositoryMock.Object,
-                _deviceRegistryListRepositoryMock.Object, _virtualDeviceStorageMock.Object,
-                _securityKeyGeneratorMock.Object, _configProviderMock.Object, _deviceRulesLogicMock.Object);
-            fixture = new Fixture();
+            this._iotHubRepositoryMock = new Mock<IIotHubRepository>();
```

Comment:
this is actually the result of code cleanup.. maybe our resharper's are configured different.